docs/topics/Opcodes_and_Status_Codes.md

```diff
@@ -6,95 +6,95 @@ All gateway events in Discord are tagged with an opcode that denotes the payload
-| 0    | Identify            | client            | Begin a voice websocket connection.                                                     |
-| 1    | Select Protocol     | client            | Select the voice protocol.                                                             |
-| 2    | Ready               | server            | Complete the websocket handshake.                                                       |
-| 3    | Heartbeat           | client            | Keep the websocket connection alive.                                                   |
-| 4    | Session Description | server            | Describe the session.                                                                   |
-| 5    | Speaking            | client and server | Indicate which users are speaking.                                                     |
-| 6    | Heartbeat ACK       | server            | Sent to acknowledge a received client heartbeat.                                       |
-| 7    | Resume              | client            | Resume a connection.                                                                   |
-| 8    | Hello               | server            | Time to wait between sending heartbeats in milliseconds.                               |
-| 9    | Resumed             | server            | Acknowledge a successful session resume.                                               |
-| 13   | Client Disconnect   | server            | A client has disconnected from the voice channel                                       |
+| Code | Name                | Sent By           | Description                                              |
+|------|---------------------|-------------------|----------------------------------------------------------|
+| 0    | Identify            | client            | Begin a voice websocket connection.                      |
+| 1    | Select Protocol     | client            | Select the voice protocol.                               |
+| 2    | Ready               | server            | Complete the websocket handshake.                        |
+| 3    | Heartbeat           | client            | Keep the websocket connection alive.                     |
+| 4    | Session Description | server            | Describe the session.                                    |
+| 5    | Speaking            | client and server | Indicate which users are speaking.                       |
+| 6    | Heartbeat ACK       | server            | Sent to acknowledge a received client heartbeat.         |
+| 7    | Resume              | client            | Resume a connection.                                     |
+| 8    | Hello               | server            | Time to wait between sending heartbeats in milliseconds. |
+| 9    | Resumed             | server            | Acknowledge a successful session resume.                 |
+| 13   | Client Disconnect   | server            | A client has disconnected from the voice channel         |
```

Comment:
I know this is outside the scope of this PR, but I may as well ask here while on the topic of documenting new functionality.

Are there any plans to look into the voice API any time soon as part of these changes?

Namely details like how no port is documented to be sent on the endpoint but you still send us port 80 in the URI (even though we have to actually use port 443); how to receive audio (you say that we can receive audio but not how to do that), etc? Or is it better to just not use the voice API at all currently?

Reply:
It's been a while since I've dug into the Voice API tbh. Last I remember was something like it still being touched very heavily and not wanting to document in case we make a lot of breaking changes. But I can bring this back up next week as a separate note.

Reply:
That would be brilliant, thanks!

Reply:
we already now send 443 in the endpoints as of a couple weeks ago